packages/dev/core/src/Animations/animatable.ts

```diff
@@ -482,11 +484,39 @@ export class Animatable {
     }
 }
 
-declare module "../scene" {
-    export interface Scene {
+declare module "../coreScene" {
+    export interface CoreScene {

```

Comment:
Just a thought, to fit the basic concept of the core scene (which should have the minimum functionality):
If we are already changing it, an esm-friendly version can be supported as well (abstracted here):

1) Define all of the functions outside of a class
2) Keep the functions defined in Scene, while users can use them outside of the scene when using the corescene.

For example (deliberately taking a public function):

```javascript
// animatable.functions.ts

export function stopAnimation(coreScene: CoreScene, target: any, animationName?: string, targetMask?: (target: any) => boolean): void {
    // MAGIC!
}

// animatable, technically stay as is except for:
import { stopAnimation } from "./animatable.functions.js";
Scene.prototype.stopAnimation = function (target: any, animationName?: string, targetMask?: (target: any) => boolean): void { 
    stopAnimation(this, target .....);
}
```

Another possibility is to not implement these helper functions at all in CoreScene. playAnimation and stopAnimation are nothing more than a helper functions. it is enough to have a reference to the animatable to stop the animation.